google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java

```diff
@@ -58,6 +59,7 @@
 
   GoogleCloudApiSelectorPanel(List<CloudLibrary> libraries) {
     this.libraries = libraries;
+    panel.setPreferredSize(new Dimension(800, 600));
```

Comment:
I just realized: can this be part of the .form?

Reply:
I wish. But for some unknown reason it renders differently when added to the form! (its wider, and the text doesn't wrap when resizing). No clue why yet.